src/extensions/extra/PDFView/PDFView.js

```diff
@@ -32,6 +32,21 @@ define(function (require, exports, module) {
                               .getExtensionPrefs("themes"),
         PDFViewTemplate     = require("text!htmlContent/pdf-view.html");
 
+    var _viewers = {};
+
+    // Track theme changes, and let any open PDF Viewers know
+    ThemePrefs.on("change", "theme", function() {
```

Comment:
Except it's not the preferences manager, it's the subtree for the theme prefs within the preferences manager.